docs/using-data-types.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[ecs-data-types]]
+=== Data Types
+
+IMPORTANT: The limitations noted in this section apply to the OSS-only distribution of the Elastic Stack. The default distribution with the Elastic license supports all data type used by ECS.
+
+Beginning with ECS 1.8.0, the ECS specification adopted additional Elasticsearch {ref-bare}/current/mapping-types.html[data types] only available in the default distribution of the Elastic Stack.
+
+OSS-only users can still benefit from normalizing data to ECS. Some licensed data types will have a fallback, but OSS-only users will not populate a field if a fallback is not available.
```

Comment:
> but OSS-only users will not populate a field if a fallback is not available

I think this is potentially misleading or confusing. The shape of the JSON event will be unaffected by whatever type is inside Elasticsearch.

Some data types under the basic license have an equivalent but less capable fallback, and some types don't have an equivalent.

So far ECS only uses basic types that have a fallback. So perhaps we can omit this whole line of discussion for now.

But if you want to cover it, it's important to note that the JSON object can still be captured 100% in Elasticsearch (e.g. a structured field aimed at populating a `histogram` field can be shoved in an `object` field). It's just that OSS users won't have a roughly equivalent way to query it; and OSS indices will conflict with basic indices, for folks migrating to basic.

But to get back to my previous point, I think I'd avoid covering this part for now.

Reply:
Fair point - I removed this part.